README.rst

```diff
@@ -37,127 +37,117 @@ the Rescorla-Wagner learning rule to these corpora.
     This package is not intended to be used under python2.
 
 
-Installation
-============
-Install with::
+Getting Started
+===============
+These instructions will get you a copy of the project up and running on your
+local machine for development and testing purposes. See deployment for notes on
+how to deploy the project on a live system.
+
+
+Prerequisites
+-------------
+You need python 3.4 or newer and git installed on your machine. We recommend to
+install Minicoda (https://conda.io/miniconda.html) before installing ``pyndl``
```

Comment:
Miniconda seems to be one of the easiest ways to get a good virtual environment for python within your userspace. What else should we suggest (going with the preinstalled python will not work for most distributions)?